src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -57,6 +57,12 @@ export const environmentProjectSchema = {
                 'The strategy configuration to add when enabling a feature environment by default',
             $ref: '#/components/schemas/createFeatureStrategySchema',
         },
+        visible: {
+            type: 'boolean',
+            example: true,
+            description:
+                'Indicates if the environment is available to be enabled for feature flags in the project',
```

Comment:
This phrase looks a bit weird, but my suggestion might be too simplistic
```suggestion
                'Indicates if the environment is visible in the project',
```